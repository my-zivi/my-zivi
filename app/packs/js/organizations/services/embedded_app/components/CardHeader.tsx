import React from 'preact/compat';
import { FunctionalComponent } from 'preact';

const CardHeader: FunctionalComponent<{ overviewSubtitle: string }> = ({ overviewSubtitle }) => (
  <div class="card-header bg-light">
    <div class="d-flex align-items-center">
      <h5>{MyZivi.translations.servicesOverview.title}</h5>
    </div>
    <h6 class="text-muted mb-0">
      {overviewSubtitle}
    </h6>
  </div>
);

export default CardHeader;
