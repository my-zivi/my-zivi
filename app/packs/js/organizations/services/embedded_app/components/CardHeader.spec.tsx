import { shallow } from 'enzyme';
import CardHeader from 'js/organizations/services/embedded_app/components/CardHeader';
import React from 'preact/compat';

describe('CardHeader', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<CardHeader overviewSubtitle={'This is my title'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
