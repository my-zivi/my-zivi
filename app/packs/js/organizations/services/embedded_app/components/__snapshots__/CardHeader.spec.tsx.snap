// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader renders correctly 1`] = `
<div
  className="card-header bg-light"
>
  <div
    className="d-flex align-items-center"
  >
    <h5>
      My Services Overview
    </h5>
  </div>
  <h6
    className="text-muted mb-0"
  >
    This is my title
  </h6>
</div>
`;
